### PR TITLE
SKR migration test improve log messages

### PR DESCRIPTION
**Pull request by wozniakjan**
**Description**

In the SKR SVCAT => BTP operator test, the descriptions and info messages were considered confusing, this should reduce some ambiguity.

Depends on https://github.com/kyma-incubator/sc-removal/pull/9

### CODE REVIEW

File: tests/fast-integration/skr-svcat-migration-test/test-helpers.js [hunk 1]
```diff
@@ -50,13 +50,8 @@ async function installBTPOperatorHelmChart(creds, clusterId) {
 }
 
 async function installBTPServiceOperatorMigrationHelmChart() {
-    const chart = "https://github.com/kyma-incubator/sc-removal/releases/download/0.3.0/sap-btp-service-operator-migration-0.3.0.tar.gz";
+    const chart = "https://github.com/kyma-incubator/sc-removal/releases/download/0.5.0/sap-btp-service-operator-migration-0.5.0.tar.gz";
```
Comment (wozniakjan) [inline tests/fast-integration/skr-svcat-migration-test/test-helpers.js:53]: this requires release of sc-removal v0.5.0

**Review by crabtree — DISMISSED** [review]

**Review by crabtree — APPROVED** [review]

**Review by voigt — APPROVED** [review]
/lgtm

**Pull request merged**